src/components/TextField/TextField.test.tsx

```diff
@@ -48,3 +50,42 @@ describe('<TextField />', () => {
     });
   });
 });
+
+describe('TextField#focus', () => {
```

Comment:
can we put this inside the main describe block instead of starting a new outer block?

Reply:
done